### ISSUE TITLE
Great package, cryptic name and description

### ISSUE THREAD

**Issue by m-sostero** [ThinkRstat/seven31#1]
The package is great and very useful, but I find the name exceedingly cryptic: reference to an R FAQ number, combination of spelled-out number and digits... 

I had used it in the past and today I struggled to find it again :sweat_smile: 

Why not use a name like *rrreveal*? 

Also, the README.md of the repo could be more transparent about what the package does (in plain English) to make it easier to find.

Thank you again for your great work!

**Comment by romainfrancois** [comment]
This was more of a week end experiment, similar to https://github.com/ThinkR-open/utf8splain

I'll think about making a more general 📦 about bits and bytes. 

In the meantime, please send a pull requester the README.
